packages/augur-ui/src/modules/app/actions/init-augur.ts

```diff
@@ -38,56 +47,75 @@ function pollForAccount(
-  const { loginAccount } = getState();
-  let accountType =
-    loginAccount && loginAccount.meta && loginAccount.meta.accountType;
-  let usingMetaMask = accountType === ACCOUNT_TYPES.WEB3WALLET;
-  if (!accountType && isGlobalWeb3()) {
-    usingMetaMask = true;
-  }
-  setInterval(() => {
-    const { authStatus, connection } = getState();
-    if (connection.isConnected) {
-      let loggedInAccount: string = undefined;
-      let loggedInAccountType: string = undefined;
-      if (!loginAccount.address) {
-        if (windowApp.localStorage && windowApp.localStorage.getItem) {
-          loggedInAccount = windowApp.localStorage.getItem('loggedInAccount');
-          loggedInAccountType = windowApp.localStorage.getItem('loggedInAccountType');
+
+  let attemptedLogin = false;
+  let intervalId = null;
+
+  async function attempLogin() {
+    const { connection } = getState();
+    if (attemptedLogin) {
+      clearInterval(intervalId);
+    } else {
+      if (connection.isConnected) {
+        attemptedLogin = true;
+
+        const loggedInAccount = windowApp.localStorage.getItem('loggedInAccount');
+        const loggedInAccountType = windowApp.localStorage.getItem('loggedInAccountType');
+
+        const showModal = (accountType) => {
+          dispatch(
+            updateModal({
+              type: MODAL_LOADING,
+              callback: () => dispatch(closeModal()),
+              message: `Syncing 📡 ${accountType} account...`,
+              showCloseAfterDelay: true,
+            })
+          );
+        }
+
+        const errorModal = () => {
+          dispatch(logout());
+          dispatch(
+            updateModal({
+              type: MODA_WALLET_ERROR,
+            })
+          );
+        };
+
+        if (loggedInAccount) {
+          try {
+            if (isGlobalWeb3() && loggedInAccountType === ACCOUNT_TYPES.WEB3WALLET) {
+              showModal(ACCOUNT_TYPES.WEB3WALLET);
+              await dispatch(loginWithInjectedWeb3());
+            }
+            if (loggedInAccountType === ACCOUNT_TYPES.PORTIS) {
+              showModal(ACCOUNT_TYPES.PORTIS);
+              await dispatch(loginWithPortis(false));
+            }
+
+            if (loggedInAccountType === ACCOUNT_TYPES.FORTMATIC) {
+              showModal(ACCOUNT_TYPES.FORTMATIC);
+              await dispatch(loginWithFortmatic());
+            }
+
+            if (loggedInAccountType === ACCOUNT_TYPES.TORUS) {
+              showModal(ACCOUNT_TYPES.TORUS);
+              await dispatch(loginWithTorus());
+            }
+          }
+          catch (error) {
+            errorModal();
+          }
         }
-      } else {
-        loggedInAccount = loginAccount.address;
-      }
-      if (!authStatus.isLogged && usingMetaMask && loggedInAccount && loggedInAccountType === ACCOUNT_TYPES.WEB3WALLET) {
-        autoLoginAccount(dispatch, loggedInAccount);
       }
     }
+  }
+
+  intervalId = setInterval(() => {
+    attempLogin();
```

Comment:
Is this spelt incorrectly on purpose? Missing t in attempt

Reply:
my mistake